here/entry.py

```diff
@@ -87,7 +87,7 @@ def display(server: ServerInterface, name: str, position: Position, dimension_st
 
 def on_info(server: PluginServerInterface, info: Info):
 	global here_user
-	if info.is_player and info.content == '!!here':
+	if info.is_player and info.content.find('!!here') != -1:
```

Comment:
If this is safe, it will save performance.
```suggestion
	if info.is_player and '!!here' in info.content:
```